docs/dbc-index.md

```diff
@@ -14,5 +14,6 @@
 * [Map](map.md)
 * [PageTextMaterial](pagetextmaterial.md)
 * [SkillLine](skillline.md)
+* [SummonProperties](summonproperties_dbc.md)
```

Comment:
wrong. In this case you should have 2 pages.

1. for the DB Structure as DB not always have all the fields that the DBC has
2. for the DBC structure and possible constant values within it

Reply:
I know, but in this case DB matches the dbc.

Reply:
fine